test/tests/preconditioners/fdp/tests

```diff
@@ -4,6 +4,13 @@
     input = 'fdp_test.i'
     exodiff = 'out.e'
     max_parallel = 1
-    deleted = '#5153'
```

Comment:
Are you resolving this ticket? Now that you are re-enabling the test we should probably close it.

Reply:
I did not do anything. FDP just works fine. Not sure why it is deleted at the first place.